components/table/src/data-table/table-elements/table-data-cell/table-data-cell.js

```diff
@@ -5,6 +5,12 @@ import PropTypes from 'prop-types'
 import React, { forwardRef } from 'react'
 import styles from './table-data-cell.styles.js'
 
+const rtlCorrespondingAlignments = {
```

Comment:
here it also seemed like we would want to assume that the physical properties we have on this component for alignment should actually behave as logical properties. Though I guess we could also add new options like `inline-start`, `inline-end` for alignment?